src/Sulu/Bundle/ContentBundle/Resources/config/routing_api.yml

```diff
@@ -11,6 +11,11 @@ sulu_content.node.seo:
     parent: sulu_content.node
     resource: Sulu\Bundle\ContentBundle\Controller\SeoController
 
+sulu_content.node.versions:
```

Comment:
Should be possible with [custom route loaders](http://symfony.com/doc/current/routing/custom_route_loader.html).